src/sql/workbench/api/common/extHostQueryEditor.ts

```diff
@@ -54,15 +54,15 @@ export class ExtHostQueryEditor implements ExtHostQueryEditorShape {
 
 	public $registerQueryInfoListener(providerId: string, listener: azdata.queryeditor.QueryEventListener): void {
```

Comment:
You should remove the providerId here too since it's no longer being used

Reply:
@gupta1 it looks like removing this parameter isn't a back-compat issue since the only call is from `azdata.queryeditor`.

Reply:
Karl mentioned that this method signature is used by other extensions so that's why keeping it as it is.